javascript-forms/form-values.js

```diff
@@ -0,0 +1,15 @@
+var $submit = document.getElementById('contact-form');
+function logSubmit(event) {
+  event.preventDefault();
+  var $name = $submit.elements[0];
+  var $email = $submit.elements[1];
+  var $text = $submit.elements[2];
```

Comment:
While this works it is a bit risky and unclear which element is which. A better approach would be to use the inputs name attribute to get the value: `$submit.elements.email.value`